reference/UiTPAS.v2.json

```diff
@@ -1148,7 +1148,7 @@
             },
             "in": "query",
             "name": "dateOfBirthTo",
-            "description": "Include only passholders with a dateOfBirth of this value (including) or older"
+            "description": "Include only passholders with a dateOfBirth of this value (including) or older. Format yyyy-mm-dd e.g. 2003-01-01"
```

Comment:
You can make this a regex and make it part of the schema to be 100% safe. (`"format": "date"` accepts any valid ISO-8601 date/time notation if I'm not mistaken)

Reply:
See https://swagger.io/docs/specification/data-models/data-types/#pattern

Reply:
An example of a regex could be `^\d{4}-\d{2}-\d{2}$`

Reply:
Nevermind ignore my comments, `"format": "date"` only accepts dates in the format that you described in the description. 👍